### ISSUE TITLE
Install on Mac (Guide readme)

### ISSUE THREAD

**Issue by imanhn** [opengisch/OSGeo4A#72]
Hi,
In section titled "MacOS Specific instructions"
the part that suggest to execute `brew tap caskroom/versions` raises an error saying :

> Error : caskroom/versions was moved. Tap homebrew/cask-versions instead.

So it should be replaced with `brew tap homebrew/cask-versions`.

and the next line `(brew cask install java8)` raises the following error

> Error: Cask 'java8' is unavailable: No Cask with this name exists.

So it should also be replaced with `brew cask install adoptopenjdk/openjdk/adoptopenjdk8` as well.

I was going to edit `Readme.md` and request a post but I couldn't do that (there is no edit on the page Readme.md)

Hope that helps someone and thanks for this awesome repo.

Cheers,
Iman

TLDR:
Please edit Readme.md with these :

`brew tap homebrew/cask-versions`
`brew cask install adoptopenjdk/openjdk/adoptopenjdk8`

**Comment by m-kuhn** [comment]
Thanks. Are you able to edit here and open a pull request?

https://github.com/opengisch/OSGeo4A/edit/master/README.md

**Comment by imanhn** [comment]
Mattias,
Sorry, for some reason I completely forgot to fork and make a pull request, I am doing it now.
Thanks for the hint.
